backend/src/database/migrations/20221129065356-drop-email-message-tx-message-id-column.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+
+module.exports = {
+  up: async (queryInterface, _) => {
+    await queryInterface.removeColumn(
+      'email_messages_transactional',
+      'message_id'
+    )
+    // removing this column will lose data irreversibly
```

Comment:
Are we storing data in this column? If so, we need to remove the code that's setting this column's value too

Reply:
Btw next time for comment like this, can u make a github comment instead of a code comment? This comment doesn't provide any extra value for the codebase

Reply:
yesser I thought I was just following this 😛

<img width="794" alt="image" src="https://user-images.githubusercontent.com/67887489/205537999-2777b946-e571-478b-9496-dd94d66e7835.png">

Reply:
> > Are we storing data in this column? If so, we need to remove the code that's setting this column's value too
> 
> Ya I was just confused why we're storing data in this column if we don't need the data... will poke around and investigate this later (don't know whether we should put this in the sprint tasks?)

Yea probably give yourself a point for this